core/steam_client.gd

```diff
@@ -276,6 +338,61 @@ func save_steamcmd_session() -> void:
 	logger.debug("Successfully saved steamcmd session to", tokens_save_path)
 
 
+## Save the given accounts (e.g. {"shadowapex": {"SteamID": 1234}}) as an entry
```

Comment:
Maybe use a fake username in the comments. E.g.{"steamyguyx": {"SteamID": 1234}})

Reply:
I was close to changing this to: `{"definitelynotpastaq": ...}` :stuck_out_tongue: